src/guides/using-final-ik.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: docs
+title: Using Final IK
+---
+# Using Final IK
+
+Final IK is the most popular Unity asset for adding realistic IK to player controllers in your game. This guide will demonstrate how to create a multiplayer player controller in Normcore using Final IK.
+
+When you're finished, you'll end up with a multiplayer game that looks like this:
+
+<video src="./creating-a-player-controller/sphere-player-mouse-jump-6.mp4" width="100%" controls></video>
+(Video demonstrating the steps above)
+
+We'll start by creating a singleplayer player controller from scratch with Final IK. (TODO: Rewrite the next sentence once the guide is complete) We'll get the controls to feel nice, and then we'll make it multiplayer. If you're only interested in the multiplayer part, skip to [making it multiplayer](#making-it-multiplayer).
+
+## Creating a player controller
+
+Before we can utilize Final IK, we'll need a player controller that allows us to move the player with WASD keys. Create a copy of the Blank Scene Template scene included in the `Normal/Examples` folder. Save it as "Final IK Player Controller".
+
+Let's start by creating a game object to represent the player. Create an empty game object in the scene named "Player" and add Normcore's built-in **Player** component.
```

Comment:
The Player Transform needs to be reset so it sits on the floor/ is centered on the camera for the blank scene template.

Reply:
Fixed in [c8d696f](https://github.com/NormalVR/Normcore-Documentation/pull/16/commits/c8d696fa82d5caebaf36a23ae688e2ffafcaefc4).